server/datastore/mysql/migrations/tables/20240709183940_AddTimeoutColumnToScriptResults.go

```diff
@@ -0,0 +1,38 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240709183940, Down_20240709183940)
+}
+
+// At the time of this migration, all script timeouts are
+// hardcoded to 300 seconds. This migration adds a timeout
+// column to the host_script_results table to allow for
+// custom timeouts.
+func Up_20240709183940(tx *sql.Tx) error {
+	stmt := `
+	ALTER TABLE host_script_results
+	ADD COLUMN timeout INT DEFAULT NULL;
```

Comment:
Why do we need this column? Just for info/debug?

Reply:
Have you considered making this column `NOT NULL` and use 0 as default?

Reply:
because timeout values are dynamic through agent config, the timeout value could differ from result to result.  storing it ensures that we report the correct timeout value that was present on the agent when the script was run.

i usually try to avoid nullable values, but in this case timeout is `null` until the agent reports back with the timeout.  (rows are created on script request and null values are updated when the agent sends the script result back).  `0` is a sensible default, but i want to keep consistent with the update pattern for other columns.